app/webpacker/stylesheets/application.scss

```diff
@@ -11,3 +11,9 @@ $button-colour: #00823b;
 .text-decoration-underline-dotted {
   text-decoration: underline dotted;
 }
+
+.course-table {
+  [data-qa="courses-table__course"] {
```

Comment:
We shouldn't use `data-qa` tags for CSS; they're meant for automation. We should add another normal class for this.